cmd/virt-launcher/readserial.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
```

Comment:
I'm missing something here... 
why do we need to introduce python into KubeVirt?
Can we write whatever this script is doing in go?
It's not difficult to `select` in go...